content/en/blog/2023-12-07-flux-release-around-the-corner/index.md

```diff
@@ -0,0 +1,24 @@
+---
+author: kingdonb
+date: 2023-12-07 00:00:00+00:00
+title: Flux release is just around the corner!
+description: "Flux v2.1.0: New monitoring setup with support for custom metrics, performance improvements and much more!"
+url: /blog/2023/12/flux-release-around-the-corner/
+tags: [announcement]
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+Now that Flux has reached General Availability, there are only 3 releases per year. So, the maintainer team has been hard at work to create this Flux v2.2.0 release, which is slated to come out next week! Keep your eye out!
```

Comment:
The release docs are stating that are at least 3 release per year.

Reply:
> Now that Flux has reached General Availability, there are planned 3 minor releases per year.

If there's another wording that's less awkward but still precise and accurate enough, then I'll use it